topics/perl/index.md

```diff
@@ -0,0 +1,12 @@
+---
+aliases: perl5, perl
+created_by: Larry Wall
+display_name: Perl
+github_url: https://github.com/Perl/perl5
+released: December 18, 1987
+short_description: Perl 5 is a highly capable and feature-rich programming language.
+topic: perl
+url: perl.org
+wikipedia_url: https://en.wikipedia.org/wiki/Perl
+---
+Perl 5 is a highly capable and feature-rich programming language. Perl 5 runs on over 100 platforms ranging from portables to mainframes and everything in between. The can be used for almost al facets of programming from rapid prototyping and large scale development projects.
```

Comment:
A few typos and stylistic edits, I'd suggest:

> Perl 5 runs on over 100 platforms ranging from portables to mainframes. The can be used for almost all facets of programming, from rapid prototyping and large-scale development projects.

Reply:
@jonasbn could you make this edit before I merge? ☝️

Reply:
Done